tests/test_cli.py

```diff
@@ -1,72 +0,0 @@
-#
-# This source file is part of the EdgeDB open source project.
-#
-# Copyright 2019-present MagicStack Inc. and the EdgeDB authors.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
-
-import edgedb
-
-from edb.testbase import server as tb
-
-
-class TestCLI(tb.ConnectedTestCase, tb.OldCLITestCaseMixin):
```

Comment:
Is this removal intentional? The commit message doesn't mention it.

Reply:
The old CLI is slated for removal and I couldn't be bothered to spend time to fix something that's going to get removed very soon anyway.